plugins/inputs/procstat/README.md

```diff
@@ -26,8 +26,9 @@ Processes can be selected for monitoring using one of several methods:
   # pattern = "nginx"
   ## user as argument for pgrep (ie, pgrep -u <user>)
   # user = "nginx"
-  ## Systemd unit name
+  ## Systemd unit name, supports globs when include_systemd_children is set to true
```

Comment:
Hi Sebastian, the sample config in procstat.go has include_systemd_children commented out with its default value. It doesn't show up here in the readme. Could you make them match in a new PR?

Reply:
Good catch, new pr to fix it: https://github.com/influxdata/telegraf/pull/9553